docs/transpiling.md

```diff
@@ -31,10 +33,27 @@ If you are using [ts-node](https://github.com/TypeStrong/ts-node):
 
 #### ESM
 
-For ESM projects, you can use `ts-node`'s ESM loader and then `import` your TypeScript files:
+For ESM projects, you can use `ts-node`'s ESM loader and then import your TypeScript files. Doing this will require setting an environment variable. the cleanest way to do this is to include the [cross-env](https://www.npmjs.com/package/cross-env) package with `npm i -D cross-env`
+
+Then change your npm test script invocation in the package.json file to:
+
+```json
+{
+  "scripts: {
+    "test": "cross-env NODE_OPTIONS=\"--loader ts-node/esm\" cucumber-js"
+  }
+}
+```
+
+To use ESM modules set the `"type": "module"` in the package.json file.
 
-```shell
-$ NODE_OPTIONS="--loader ts-node/esm" cucumber-js --import 'step-definitions/**/*.ts'
+Your cucumber.js file will need to be an ESM module and contain the following at a minimum:
+
+```javascript
+export default {
+  requireModule: ['ts-node/register'], 
```

Comment:
This doesn’t seem right? require-module doesn’t affect import

Reply:
I think this can just be removed.

Reply:
It needs to be reworded heavily and I've done so. If you read that header as applying to all ESM projects others will too. That sub header is level 4 and it was meant to cover "with TS-Node to load ES modules"